chia/wallet/puzzles/pool_member_innerpuz.clvm

```diff
@@ -48,16 +50,16 @@
 
   ; main
 
-  (if spend_type
-    (list (list AGG_SIG_ME OWNER_PUBKEY (sha256tree key_value_list ()))
+  (if pool_reward_height
```

Comment:
This if is backwards!  You want to do the absorb case if the height is passed in.